pkg/bff/models/v1/registration.go

```diff
@@ -1,14 +1,77 @@
 package models
 
 import (
-	"errors"
 	"fmt"
 	"strings"
 
+	"github.com/trisacrypto/trisa/pkg/ivms101"
 	pb "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+const (
+	// Basic Details Fields
+	FieldWebsite          = "website"
+	FieldBusinessCategory = "business_category"
+	FieldVASPCategories   = "vasp_categories"
+	FieldEstablishedOn    = "established_on"
+	FieldOrganizationName = "organization_name"
+
+	// Legal Person Entity Fields
+	FieldEntity = "entity"
+
+	// Contacts Fields
```

Comment:
@pdeziel should we use these field constants to make sure things are correctly spelled?

Reply:
Sure, probably a good idea. Thanks for adding them!